frontend/packages/kubevirt-plugin/src/style.scss

```diff
@@ -6,7 +6,6 @@
 @import "~patternfly/dist/sass/patternfly/variables";
```

Comment:
these are only the dependencies neeeded for the react-console, right? It is probably the only component which uses pf3.

are all of these imports needed?

Reply:
I tried to remove them but they appeared to be needed by console

Reply:
ok, can you please write a comment there describing these dependencies? So people know why they are here in the future.